buildchow/etc/todirformat.sh

```diff
@@ -0,0 +1,14 @@
+FILES=$1/*_$2
```

Comment:
going to update it to be a bit smarter.

but in case you want to use it now you call it as:

`todirformat generated master` or `todirformat generated 4.3.mars` and it will make a `generated-old` folder following jbdevstudio-ci "layout"